website/docs/d/nsxt_app_port_profile.html.markdown

```diff
@@ -27,18 +27,23 @@ data "vcd_nsxt_app_port_profile" "custom" {
 }
 ```
 
-## Example Usage 2 (Find an Application Port Profile defined by Tenant)
+## Example Usage 2 (Find an Application Port Profile defined by Tenant in a VDC Group)
 
 ```hcl
```

Comment:
why example 1 still using the old way?

Reply:
I think you had in mind the first example above. Fixed it.

Reply:
I think you had in mind the first example above. Fixed it.

Reply:
yes, you go it